File: Angular/angularcrud/src/app/models/session.ts
import { Coach } from "./coach";
import { Cours } from "./cours";

export interface Session {
    id: number;
    nom: string;
    dateHeureDebut: Date;
    dateHeureFin: Date;
    coursId: number;
    cours: Cours;
    coachId: number;
    coach: Coach;
  }

File: GymParkAppWeb/Models/Session.cs
namespace GymParkAppWeb.Models
{
    public class Session
    {
        public int Id { get; set; }
        public string Nom { get; set; }
        public DateTime DateHeureDebut { get; set; } // Ajoutez la propriété "dateHeureDebut"
        public DateTime DateHeureFin { get; set; } // Ajoutez la propriété "dateHeureFin"

        // Clé étrangère pour le Cours associé
        public int CoursId { get; set; }
        public Cours Cours { get; set; }

        // Autres propriétés selon les besoins

        public int CoachId { get; set; }
        public Coach Coach { get; set; }

    }

    }
